### ISSUE TITLE
Building from AUR fails -- Dependency "scdoc" not found

### ISSUE THREAD

**Issue by trwnh** [ErikReider/SwayNotificationCenter#107]
```
SwayNotificationCenter-0.5/meson.build:60: WARNING: Project targeting '>= 0.50.0' but tried to use feature introduced in '0.51.0': dependency.get_variable.
Found CMake: /usr/bin/cmake (3.22.3)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Run-time dependency fish found: NO (tried pkgconfig and cmake)
Found CMake: /usr/bin/cmake (3.22.3)
WARNING: CMake Toolchain: Failed to determine CMake compilers state
Build-time dependency scdoc found: NO (tried pkgconfig and cmake)

SwayNotificationCenter-0.5/meson.build:93:0: ERROR: Dependency "scdoc" not found, tried pkgconfig and cmake

A full log can be found at /home/trwnh/.cache/yay/swaync/src/build/meson-logs/meson-log.txt
==> ERROR: A failure occurred in build().
    Aborting...
 -> error making: swaync
```

**Comment by ErikReider** [comment]
Should be fixed :)

**Comment by eli-schwartz** [comment]
But surely it is only a build dependency? Meson needs it to run the manpage `custom_target()`. It need not be installed as a runtime dependency.

**Comment by ErikReider** [comment]
Fixed a5895e4670a92cb4030143ceade7f2d3fabce52a
